indico/htdocs/js/indico/modules/timetable/timetable/Draw.js

```diff
@@ -551,46 +548,123 @@ type("TimetableBlockWholeDayBase", ["TimetableBlockBase"],
      }
    );
 
+function drawBalloon(self, evt, editable) {
+    var timetableBlock = self.div.dom;
+    var entryId = self.eventData.scheduleEntryId ? self.eventData.scheduleEntryId : self.eventData.id.substring(1);
+    var params = {
+       'confId': self.eventData.conferenceId[0],
+       'timetable_entry_id': entryId,
+       'editable': editable ? 1 : 0
+    };
+    var balloonId = '#qtip-' + entryId.toString();
+
+    $(timetableBlock).qbubble({
+       id: entryId.toString(),
+       content: {
+           text: function(evt, api) {
+               $.ajax({
+                   url: build_url(Indico.Urls.Timetable.entries.balloon, params)
+               })
+               .then(function(content) {
+                   // Set the tooltip content upon successful retrieval
+                   api.set('content.text', content.html);
+                   var $content = api.elements.content;
+                   if (editable) {
+                       $duration = $content.find('.js-edit-time');
```

Comment:
you are creating a global here. maybe add a `'use strict';` at the beginning of that function?